Mathlib/Algebra/BigOperators/Group/Finset/Basic.lean

```diff
@@ -862,9 +862,9 @@ lemma prod_insert_div (ha : a ∉ s) (f : ι → G) :
 theorem prod_erase_eq_div {a : ι} (h : a ∈ s) : ∏ x ∈ s.erase a, f x = (∏ x ∈ s, f x) / f a := by
   rw [eq_div_iff_mul_eq', prod_erase_mul _ _ h]
 
-/-- A telescoping product along `{0, ..., n - 1}` of a commutative group valued function reduces to
+/-- A telescoping product along `{0, ..., n - 1}` of a commutative group-valued function reduces to
```

Comment:
Is the meaning here "valued in commutative groups", or that the function is commutative?